Sources/SKCore/FileBuildSettingsChange.swift

```diff
@@ -16,8 +16,13 @@ public enum FileBuildSettingsChange {
   /// The `BuildSystem` has no `FileBuildSettings` for the file.
   case removedOrUnavailable
 
-  /// The `FileBuildSettings` have been modified.
+  /// The `FileBuildSettings` have been modified or are newly available.
   case modified(FileBuildSettings)
+
+  /// The `BuildSystem` is providing fallback arguments which may not be correct.
+  ///
+  /// This indicates that diagnostics should be withheld for the given file.
```

Comment:
Please drop the second line - the policy (hide diagnostics) is not up to the build system, it's up to the language server.

Reply:
Done